src/diffusers/models/modeling_flax_utils.py

```diff
@@ -435,6 +436,7 @@ def from_pretrained(
             # make sure all arrays are stored as jnp.ndarray
             # NOTE: This is to prevent a bug this will be fixed in Flax >= v0.3.4:
             # https://github.com/google/flax/issues/1261
```

Comment:
I'd remove the note about a flax bug if we end up doing the conversion on jnp arrays. And indent the comments correctly.